### ISSUE TITLE
Anomaly "error with no safe_id attached: incorrect VtMeta classification." with `coqtop -emacs -time`

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#6326]
#### Version

master (d9f79d97dbc503e149cba2df1b228a94d7ac970b)

#### Operating system

Linux

#### Description of the problem

In `foo.v`, I have:
```coq
(* -*- coq-prog-args: ("-q" "-emacs" "-time") -*- *)
Ltac foo := idtac; idtac; exact I.

Goal True.
  Set Ltac Profiling.
  foo.
  Show Ltac Profile CutOff 0.
  Backtrack 5 0 0 .
  Show Ltac Profile CutOff 0.
  exact I.
Defined.
```
If I evaluate to after the first `Show Ltac Profile`, and then backtrack one step, in PG, I get this anomaly.  I get the same anomaly if I do `cat foo.v | ~/.local64/coq/coq-master/bin/coqtop -emacs -time -q`.  This works fine in 8.7.0.

**Comment by maximedenes** [comment]
#6318 does seem to fix the issue.

**Comment by ejgallego** [comment]
Thanks @JasonGross and @maximedenes for testing, in this area we are looking at lots of strange combinations and it is hard sometimes to reason about the effects patches will have.

**Comment by ejgallego** [comment]
Fixed by #6318 .
